test/web-platform-tests/run-wpts.js

```diff
@@ -41,17 +41,15 @@ checkToRun();
 let wptServerURL;
 let serverProcess;
 const runSingleWPT = require("./run-single-wpt.js")(() => wptServerURL);
-before({ timeout: 30 * 1000 }, async () => {
+const wptServerPromise = (async () => {
   const { urls, subprocess } = await startWPTServer({ toUpstream: false });
   wptServerURL = urls[0];
   serverProcess = subprocess;
```

Comment:
Arguably, this should be turned back into a `Promise.then(…)` callback, instead of an **IIAAFE** (Immediately‑invoked async arrow function expression).

Reply:
I tend to disagree. I generally prefer using `async` functions as much as possible.